main/compute_recovery_rate.jl

```diff
@@ -8,13 +8,22 @@ using Transducers
 
 function _file_path_to_nt(file_path::String, t1, threshold)
```

Comment:
```suggestion
"""
# Notes
- `simulation_height_success`: once height becomes lower than a prescribed threshold, `false`. Otherwise, `true`.
- `simulation_actual_time_success`: If a simulation time (real time) takes too long, `false`. Otherwise, `true`.
"""
function _file_path_to_nt(file_path::String, t1, threshold)
```